x-pack/legacy/plugins/maps/public/components/layer_toc_actions.js

```diff
@@ -51,7 +52,11 @@ export class LayerTocActions extends Component {
   };
 
   _renderPopoverToggleButton() {
-    const { icon, tooltipContent } = this.props.layer.getIconAndTooltipContent(this.props.zoom);
+    const { icon, tooltipContent, areResultsTrimmed } = this.props.layer.getIconAndTooltipContent(this.props.zoom);
+    const infoButton = areResultsTrimmed ? (<EuiIconTip
+      type="iInCircle"
+    />) : null;
+    const displayClassName = areResultsTrimmed ? `mapTocEntry-exceedLimit` : null;
     return (
       <EuiToolTip
```

Comment:
Truncation is broken with the addition of this Tooltip. To fix it, can you add 
```suggestion
       <EuiToolTip anchorClassName="mapLayTocActions__tooltipAnchor"
```

And then append that class to this selector:

https://github.com/elastic/kibana/blob/2a0ea9eaed406a1475ebd65249aeaf514382ab1d/x-pack/legacy/plugins/maps/public/connected_components/widget_overlay/layer_control/layer_toc/toc_entry/_toc_entry.scss#L53-L55